### PR TITLE
trigger alternative action field

### PR DESCRIPTION
**Pull request by jsilhan**

### CODE REVIEW

File: pynotify/models.py [hunk 1]
```diff
@@ -41,6 +41,8 @@ class BaseTemplate(BaseModel):
     text = models.TextField(null=True, blank=True, verbose_name=_l('text'))
     trigger_action = models.CharField(max_length=2500, null=True, blank=True,
                                       verbose_name=_l('trigger action'))
+    trigger_action_alternative = models.CharField(max_length=2500, null=True, blank=True,
+                                                  verbose_name=_l('trigger action alternative'))
```
Comment (rubickcz) [inline pynotify/models.py:45]: You forgot to generate translations, see https://django-pynotify.readthedocs.io/en/latest/contributing.html#get-started on how to do that.

Comment (jsilhan) [reply pynotify/models.py:45]: now it's there

File: example/locale/cs/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2020-12-09 13:43+0000\n"
```
Comment (matllubos) [inline example/locale/cs/LC_MESSAGES/django.po:11]: why po file is updated without change?

File: pynotify/models.py (hunk 1 above)
Comment (matllubos) [reply pynotify/models.py:45]: why is used name alternative?

Comment (matllubos) [reply pynotify/models.py:45]: @rubickcz you have no problem with name of the field? Make it sense with library naming confention. Maybe use array field with more trigger actions or custom model? Just asking

File: pynotify/models.py [hunk 3]
```diff
@@ -14,16 +13,41 @@
 
 from .config import settings
 from .exceptions import MissingContextVariableError
-from .helpers import DeletedRelatedObject, SecureRelatedObject, get_from_context, strip_html
+from .helpers import DeletedRelatedObject, SecureRelatedObject, get_from_context, json_dump_dict, strip_html
 
 
-class BaseModel(SmartModel):
+class BaseModelMeta(SmartModelBase, type):
```
Comment (matllubos) [inline pynotify/models.py:19]: why is type the ancestor?

File: pynotify/models.py [hunk 4]
```diff
@@ -36,11 +60,13 @@ class BaseTemplate(BaseModel):
         title: Title of the notification.
         text: Text of the notification.
         trigger_action: Arbitrary action performed when user triggers (i.e. clicks/taps) the notification.
+        extra_fields: Can be used to store additional fields needed in particular use case.
     """
     title = models.CharField(max_length=200, verbose_name=_l('title'))
     text = models.TextField(null=True, blank=True, verbose_name=_l('text'))
     trigger_action = models.CharField(max_length=2500, null=True, blank=True,
                                       verbose_name=_l('trigger action'))
+    extra_fields = models.TextField(null=True, blank=True, verbose_name=_l('extra fields'))
```
Comment (matllubos) [inline pynotify/models.py:69]: why not use JSONField from django==3.2 it is standard model field

**Pull request closed without merging**